### ISSUE TITLE
SooHyoung Oh lablgtk+ tutorial

### ISSUE THREAD

**Issue by Chris00** [ocaml/v2.ocaml.org#739]
These tutorials are at
- http://web.archive.org/http://plus.kaist.ac.kr/~shoh/ocaml/lablgtk2/treeview-tutorial/
- http://web.archive.org/http://plus.kaist.ac.kr/~shoh/ocaml/lablgtk2/lablgtk2-tutorial/

but no longer at their original locations:
- http://plus.kaist.ac.kr/~shoh/ocaml/lablgtk2/lablgtk2-tutorial/
- http://plus.kaist.ac.kr/~shoh/ocaml/lablgtk2/treeview-tutorial/

Since they are [of interest to some people](https://sympa.inria.fr/sympa/arc/caml-list/2015-12/msg00022.html), we may want to port them to the site not to loose them (trying to contact the author if possible).

**Comment by cwfoo** [comment]
Perhaps you can just replace those broken links with links to the latest non-404 archived versions of those tutorials.

Latest non-404 copies can be found using the Wayback Machine API:

* https://archive.org/wayback/available?url=plus.kaist.ac.kr/~shoh/ocaml/ocamllex-ocamlyacc/ocamllex-tutorial/
* https://archive.org/wayback/available?url=plus.kaist.ac.kr/~shoh/ocaml/ocamllex-ocamlyacc/ocamlyacc-tutorial/
* https://archive.org/wayback/available?url=plus.kaist.ac.kr/~shoh/ocaml/lablgtk2/lablgtk2-tutorial/

**Comment by pmetzger** [comment]
I think linking to archive.org isn't optimal. It would be better to put the tutorials on the site itself.

**Comment by cwfoo** [comment]
@pmetzger I agree. It's just that it's somewhat inappropriate to keep broken links around for so long.

**Comment by pmetzger** [comment]
So why not help by getting in touch with the original author, and making a pull request of new web pages with the original information, cleaned up to match the site style?

**Comment by Chris00** [comment]
> So why not help by getting in touch with the original author, and making a pull request of new web pages with the original information, cleaned up to match the site style?

Please proceed! :-)

**Comment by ohama** [comment]
I'm SooHyoug.
If you have any comments, please contact me, through ohama100@gmail.com.

**Comment by ohama** [comment]
> I'm SooHyoug.
> If you have any comments, please contact me, through [ohama100@gmail.com](mailto:ohama100@gmail.com).

SooHyoung Oh.

**Comment by pmetzger** [comment]
@ohama Would it be okay for ocaml.org to publish your tutorial?

**Comment by ohama** [comment]
I'm converting the old document to markdown and moving to github.
See https://ohama.github.io/ocaml/

I'm on vacation until 28th May
so I'm planning to open the new ocaml related tutorial page after then.

And it's ok to publish in there.
How can I help you?

Thanks.



2019년 5월 15일 (수) 오전 2:51, Perry E. Metzger <notifications@github.com>님이 작성:

**Comment by ohama** [comment]
Current draft in https://ohama.github.io/ocaml is not final,
so don’t use the current web address until I open to ocaml community.
I’m thinking it may be better to separate each tutorial documents.

나의 iPad에서 보냄

2019. 5. 15. 오전 2:51, Perry E. Metzger <notifications@github.com> 작성:

**Comment by Chris00** [comment]
On 17 May 2019 at 00:52 CEST, SooHyoung Oh <notifications@github.com> wrote:
>
> […] And it's ok to publish in there.
> How can I help you?

Once everything is converted to markdown, we can integrate it in ocaml.org (or you may want to make a PR).  Thanks for your help (and enjoy your holidays!).

**Comment by ghost** [comment]
At present, this might fit best in ocamlverse?
